backend/services/esweService.ts

```diff
@@ -10,8 +10,8 @@ import {
   LearnerLatestAssessment,
   LearnerProfile,
   LearnerEducation,
-  LearnerNeurodivergence,
 } from '../api/curious/types/Types'
+import { createFlaggedContent } from '../shared/neuroDivergenceHelper'
```

Comment:
Not sure why this is here?!

Reply:
_createFlaggedContent_ is now a shared resource, it's used in all methods calling the curiousApi in esweService.ts and is also used in prisonerPersonal.ts